primitives/src/constants.rs

```diff
@@ -219,12 +219,10 @@ pub mod did {
 
 pub mod treasury {
 	use crate::{Balance, BlockNumber};
-	use frame_support::PalletId;
 
 	use super::{BLOCKS_PER_YEAR, KILT};
 
 	pub const INITIAL_PERIOD_LENGTH: BlockNumber = BLOCKS_PER_YEAR.saturating_mul(5);
 	const YEARLY_REWARD: Balance = 2_000_000u128 * KILT;
 	pub const INITIAL_PERIOD_REWARD_PER_BLOCK: Balance = YEARLY_REWARD / (BLOCKS_PER_YEAR as Balance);
-	pub const TREASURY_PALLET_ID: PalletId = PalletId(*b"kilt/tsy");
```

Comment:
Q: Why did you move this out of constants?

Reply:
The IDs should all be at the same place so that you can spot duplicates quickly.
We could have them in the primitives constants. But IMHO this makes it only more complicated. They are mostly used in a single place. It doesn't matter if they differ between peregrine and spiritnet so I don't see the point in moving them to the primitives.

I'm also not sure if it's a good idea to have a fixed prefix like `kilt/`. effectively there are only three letters left to change?

Reply:
I moved it back to the primitives.

Reply:
143923e